### ISSUE TITLE
Test Tubes Page: No indication of which visualization is currently displayed.

### ISSUE THREAD

**Issue by mhuyck** [greenelab/adage-server#325]
**Priority 2: Minor usability problem: fixing this should be given low priority**
Might not be relevant if the volcano plot has a separate page

**Comment by mhuyck** [comment]
Both heatmap and volcano plot will work the same after #62 is finished. Let's review this question after that other work is done.
